### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by jeacott** [kuujo/vertigo#45]

**Comment by kuujo** [comment]
Yes this project is largely dead at the moment. Having been designed for Vert.x 2, there was previously some work to move it to Vert.x 3. But while there still remains a lot of interest from various users in maintaining it, I don't have the time to actively develop this project because of my work on [Atomix](http://github.com/atomix/atomix) and [Copycat](http://github.com/atomix/copycat) and my day job. All of my efforts in open source will be going into the Atomix projects for the foreseeable future.

That said, some other users have taken on porting the project to Vert.x 3. However, I don't have the links immediately handy and I'm not sure how far those efforts ever got.

I'm going to keep this open for other users browsing the repo.
